### ISSUE TITLE
Please allow PVC selection via an existingClaim for persistent storage instead of dynamic PVC provistioning

### ISSUE THREAD

**Issue by HummingMind** [prometheus-operator/prometheus-operator#6193]
Hello, would it be possible to allow PVC selection via an existingClaim for persistent storage instead of dynamic PVC provistioning for Promethes/AlertManager CRDs?

My scenario:

I have an existing PV set up mapped to an Azure Premium file share. Each file share has a minimum of 100GB, so i only need one for Grafana, Prometheus, and AlertManager persistent storage. I can use the existingClaim field for the Grafana chart. The standalone Prometheus community helm chart has support for this as well. However, it is my understanding that Prometheus Opertator does not support this and tries to provision a new PVC dynamically every time (which gets expensive in Azure with 100GB each for Premium storage). The problem with that is the 1-to-1 mapping for PVCs to PVs. If I already have a PVC created by the Prometheus CRD, I can't use the same PVC for AlertManager. Or if I have a manually provisitoned PVC already created for Grafana or something else, that is bound to this file share, then I can't point Prometheus/AlertManager to use the existing PVC.

Thank you!

**Comment by simonpasquier** [comment]
Isn't your request already documented by https://prometheus-operator.dev/docs/operator/storage/#manual-storage-provisioning?

**Comment by HummingMind** [comment]
I was getting an error in AKS (Azure k8s Service) that specifying PVC by a label selector wasn't suported (so the example in the docs you linked wasn't working).

I eventually got it to work by using volumeName (that is listed in the API spec) instead of the lavel selector. But then I switched to a different storage type altogether, so I can use dynamically provisioned PVs.

I personally no longer need this feature, so I'll close the ticket.

Thank you!
